.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,8 +1,31 @@
-**Does this address an existing feature request? If so, link it here:**
-<!--Provide the ticket number, and the source. If it is a GitHub ticket, then #ticketNum will be sufficient. A WowAce ticket should include a URL.-->
-
-**Is there any additional work that needs to be done?**
-<!--Make a list here of things that you know are still incomplete in this submission
-E.g.:
-- [ ] Fix regression in doohickey 1
-- [x] Ensure that widget 483 can access this feature-->
+# Description
+
+Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.
+<!-- If it is a GitHub ticket, then #ticketNum will be sufficient. A WowAce ticket should include a URL. -->
+Fixes # (issue)
+
+## Type of change
+
+Please delete options that are not relevant.
+
+- [ ] Bug fix (non-breaking change which fixes an issue)
+- [ ] New feature (non-breaking change which adds functionality)
+- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
+- [ ] This change requires a documentation update
+
+# How Has This Been Tested?
+
+Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration
+
+- [ ] Test A
+- [ ] Test B
+
+# Checklist:
```

Comment:
You can add a markdown comment to these checklists to mention that these can be checked off after the pull request is submitted, in case the person wants to open discussion before they’re completely ready to submit